back/src/Guide/Database/Schema.hs

```diff
@@ -2,21 +2,23 @@
 {-# LANGUAGE QuasiQuotes       #-}
 
 
-module Guide.Db.Schema
+--
+-- | Schemas to create table for guide database
```

Comment:
This PR introduces three different styles for module comments. This is inconsistent.

The comment style we should use is:

```haskell
-- | Foo bar baz.
-- Blah blah blah.
module Foo where
```